client/landing/stepper/declarative-flow/internals/steps-repository/importer-migrate-message/index.tsx

```diff
@@ -160,7 +160,8 @@ const ImporterMigrateMessage: Step = ( { navigation } ) => {
 									),
 									{
 										email: user?.email,
-										webSite: fromUrl,
+										// Strip protocol and trailing slash.
+										webSite: fromUrl.replace( /(^\w+:|^)\/\//, '' ).replace( /\/$/, '' ),
```

Comment:
I wonder if it might be preferable to use `urlToSlug` or `withoutHttp` here? 

https://github.com/Automattic/wp-calypso/blob/7e5f54c29580df8e97c1443c7f95b2011a8cdf80/client/lib/url/http-utils.ts#L6-L34

Reply:
Thanks for letting me know about these. `urlToDomainAndPath` looks like it does exactly what's needed here. 🥳